src/services/goals.js

```diff
@@ -50,7 +50,7 @@ const OPTIONS_FOR_GOAL_FORM_QUERY = (id, recipientId) => ({
   attributes: [
     'id',
     'endDate',
-    'name',
```

Comment:
this wont work, that is building the command for sequelize, it does not work like that.

Reply:
please remove all the ''.trim()

Reply:
👍